internal/web_api.go

```diff
@@ -82,14 +87,30 @@ func (a *api) Call(action string, reqPayload interface{}, respPayload interface{
 		return fmt.Errorf("couldn't create new http request: %v", err)
 	}
 
-	rawBody, err := json.Marshal(reqPayload)
-	if err != nil {
-		return err
-	}
-	req.GetBody = func() (io.ReadCloser, error) {
-		return ioutil.NopCloser(bytes.NewReader(rawBody)), nil
+	if len(opts) > 0 && opts[0].Method == http.MethodGet {
+		req.Method = opts[0].Method
+		qs, err := query.Values(reqPayload)
+		if err != nil {
+			return err
+		}
+		if req.URL.RawQuery != "" {
+			for key, values := range req.URL.Query() {
+				for _, value := range values {
+					qs.Add(key, value)
+				}
+			}
+		}
+		req.URL.RawQuery = qs.Encode()
```

Comment:
won't it be possible to append `qs.Encode()` to `RawQuery` if the latter is not empty?

Reply:
It's possible.